xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
+      You can determine the device name of the terminal you are currently
+      logged into by running the <literal>tty</literal> command. Be careful
+      not to do this in a virtual terminal like via SSH or in a graphical
+      session (device names <filename>/dev/pts/&lt;N&gt;</filename>) but only
+      from an actual login terminal reachable via
+      <keycombo><keycap function="alt"/><keycap>F&lt;N&gt;</keycap></keycombo>.
+     </para>
+    </note>
+   </sect2>
+
+   <sect2 xml:id="sec-sec-prot-retrict-root-graphical">
+    <title>Restricting Graphical Session Logins</title>
+
+    <para>
+     On security sensitive systems like servers it is best not to run a
+     graphical user environment at all. If you require a graphical login
+     anyways then it is even more so important that the &rootuser;
+     user cannot login in a graphical session. Graphical programs are often
+     not designed to be run as &rootuser; and are more likely to
+     contain security issues than console programs.
+    </para>
+
+    <para>
+     To deny the &rootuser; user login to graphical sessions you
+     can apply the same basic steps as outlined in
+     <xref linkend="sec-sec-prot-restrict-root-tty"/>.
+     You only need to also add the <literal>pam_securetty</literal> module to
+     the PAM stack file belonging to the display manager, for example
+     <filename>/etc/pam.d/gdm</filename> for GDM. The graphical session also
+     runs on a TTY device, <literal>tty7</literal>, by default. Therefore,
+     when you restrict &rootuser; logins to
+     <literal>tty1</literal>, then a &rootuser; login in the
+     graphical session will be denied.
+    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-sec-prot-restrict-root-ssh">
+    <title>Restricting SSH Logins</title>
+    <para>
+     By default the &rootuser; user is also allowed to login
+     into a machine remotely via the SSH network protocol (if the SSH port is
+     not blocked by the firewall). To restrict this possibility the following
+     change of the OpenSSH configuration needs to be performed:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>Edit <filename>/etc/ssh/sshd_config</filename> and adjust the following parameter:</para>
+      <screen>PermitRootLogin no</screen>
+     </listitem>
+     <listitem>
+      <para>Restart the <literal>sshd</literal> service to make the changes effective:</para>
+      <screen>systemctl restart sshd.service</screen>
+     </listitem>
+    </itemizedlist>
+    <note>
+     <para>
+      Using the PAM module <literal>pam_securetty</literal> is not suitable in
+      case of OpenSSH, because SSH logins not necessarily go through the PAM
+      stack during the auth phase (e.g. when using SSH public key authentication).
+      Also an attacker could be able to differentiate between a wrong password
+      and a successful login that was only rejected later on by policy.
```

Comment:
This is not an action, but information. Is there anything the user has to do?

Reply:
No, that's why this is a `<note>` ;)

Reply:
Ok, fine with me. :-)